dbtsl/api/graphql/client/asyncio.py

```diff
@@ -49,6 +49,7 @@ def __init__(
         auth_token: str,
         url_format: Optional[str] = None,
         timeout: Optional[Union[TimeoutOptions, float, int]] = None,
+        client_partner_source: Optional[str] = None,
```

Comment:
I'd prefer if we moved this to a property and out of the constructor to reduce the cognitive overload of regular users, since 99% of people won't need to set this param.